package.json

```diff
@@ -2,15 +2,15 @@
   "name": "macrostrat-api",
   "version": "3.0.0-dev",
   "description": "The API for SCIENCE",
-  "main": "server.js",
+  "main": "server.ts",
   "repository": {
     "type": "git",
     "url": "https://github.com/UW-Macrostrat/macrostrat-api.git"
   },
   "scripts": {
     "start": "nodemon --watch v2 --exec ts-node --transpileOnly server.ts",
     "serve": "ts-node --transpileOnly server.ts",
-    "test": "mocha ./v2",
+    "test": "mocha --require ts-node/register/transpile-only './v2test/test/test.js'",
```

Comment:
Can you add a npm script that both transpiles with `tsc` and runs `mocha` so the whole testing process can run in one shot?